_release/bat/base_bat/base_bat_test.go

```diff
@@ -17,16 +17,32 @@
 package basebat
 
 import (
+	"bytes"
 	"context"
 	"flag"
+	"io"
+	"io/ioutil"
+	"net/http"
 	"os"
 	"testing"
 	"time"
 
 	"github.com/01org/ciao/bat"
+	"golang.org/x/crypto/ssh"
+	"gopkg.in/yaml.v2"
 )
 
 const standardTimeout = time.Second * 300
+const vmCloudInit = `---
+#cirros cloud-config
+users:
+  - name: ciaouser
+    geocos: CIAO Test User
+    lock-passwd: false
+    passwd: abc123
```

Comment:
This is not correct.  The password needs to be encoded, e.g., 

$6$rounds=4096$ytTyxEFgRvD$R9Z28ondJny4zcS6qvQBw7qqy3qF2TlSmdcMU/VIv88S6gzWqzqHQzcTphGaBS6E6RqXeg5RcAJzL0m9kmIM4/

which I get from

mkpasswd --method=SHA-512 --rounds=4096